webapp/src/features/intersections/data-selector/data-selector-edit-form.tsx

```diff
@@ -173,9 +167,10 @@ export const DataSelectorEditForm = (props: {
   return (
     <form onSubmit={formik.handleSubmit} {...other}>
       <Card>
-        {/* <CardHeader title="Edit Configuration Parameter" /> */}
-        <Divider />
         <CardContent>
+          <Typography noWrap variant={'h6'} sx={{ mt: -1, mb: 4 }}>
+            Query
+          </Typography>
           <Grid2 container spacing={3}>
             <Grid2 size={{ md: 6, xs: 12 }}>
               <FormControl fullWidth error={Boolean(formik.touched.intersectionId && formik.errors.intersectionId)}>
```

Comment:
Let's add a backlog item for removing the additional Intersection ID selector, now that the intersection ID selector is moved into the same page

Reply:
Added a backlog item for this